opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -820,7 +823,10 @@ mod tests {
             (OTEL_BSP_EXPORT_TIMEOUT, Some("2046")),
         ];
         temp_env::with_vars(env_vars.clone(), || {
-            let builder = BatchSpanProcessor::builder(TestSpanExporter::new(), runtime::Tokio);
```

Comment:
Are there more places `TestSpanExporter` is used? If none, could we remove that struct itself?

Reply:
Only used here, removed the struct.